drivers/wifi/infineon/airoc_wifi.h

```diff
@@ -34,11 +62,25 @@ struct airoc_wifi_data {
 	uint8_t frame_buf[NET_ETH_MAX_FRAME_SIZE];
 };
 
+union airoc_wifi_bus {
+#if defined(CONFIG_AIROC_WIFI_BUS_SDIO)
+	const struct device *bus_sdio;
+#elif defined(CONFIG_AIROC_WIFI_BUS_SPI)
```

Comment:
I don't think this is exclusive.
Although this is probably not a common case, it is best to assume that there are two devices connected via sdio and spi.

Reply:
On the one hand, the Infineon AIROC driver only supports a single device.  On the other hand ... if they ever change that, fixing this will mean the airoc_wifi_bus definition will not have to change.  Done.